tui/paraprogress/paraprogress.go

```diff
@@ -168,7 +168,10 @@ func (md *ParaProgress) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 
 	if complete == len(md.processes) {
 		md.quitting = true
-		return md, tea.Sequence(tea.Batch(cmds...), tea.Quit)
+		// This line will disable linting for the whole file, be careful
+		//nolint:staticcheck
+		//lint:ignore SA1019 using Sequence is currently breaking inside bubbletea
+		return md, tea.Sequentially(tea.Batch(cmds...), tea.Quit)
```

Comment:
This issue arises from upgrading bubbletea which exposes a new API. Since using this API throws a nil pointer exception and instead of ignoring the underlying issue and using the older now deprecated API by ignoring all issues in the file [a separate issue in itself](https://github.com/golangci/golangci-lint/issues/741), we should address the nil pointer directly. Taking a look at how the new API from bubbletea works shows that the nil will be returned by `tea.Batch` if the list of `cmds` is empty. So we can check for this return type and simply `tea.Quit` if this is the case:

```go

		batch := tea.Batch(cmds...)
		if batch == nil {
			return md, tea.Quit
		}

		return md, tea.Sequence(batch, tea.Quit)
```

Please adapt the above explanation text in the git commit message which fixes this issue.

Reply:
[Ideally the `nil` `cmd` should actually be checked by bubbletea](https://github.com/charmbracelet/bubbletea/issues/639).